src/shopify.ts

```diff
@@ -70,6 +75,33 @@ export const deployTheme = async (
   });
 };
 
+export const duplicateLive = async (SHOPIFY_AUTH: shopifyAuth, id: number): Promise<void> => {
+  !fs.existsSync(`./.shopify-tmp/`) && fs.mkdirSync(`./.shopify-tmp/`, { recursive: true });
+  await themeKit.command(
+    "download",
+    {
+      password: SHOPIFY_AUTH.password,
+      store: SHOPIFY_AUTH.storeUrl,
+      live: true,
+      noIgnore: true,
+      dir: "./.shopify-tmp/",
+    },
+    { logLevel: "all" }
+  );
+  await themeKit.command(
+    "deploy",
+    {
+      password: SHOPIFY_AUTH.password,
+      store: SHOPIFY_AUTH.storeUrl,
+      themeId: id,
+      noIgnore: true,
+      dir: "./.shopify-tmp/",
+    },
+    { logLevel: "all" }
+  );
+  fs.rmdirSync("./.shopify-tmp/", { recursive: true });
+};
```

Comment:
Great addition, something I've been wanting to add for a while! Can you add some documentation into the readme?

Reply:
Yeah, i'll add in some free time soon ;)